addon-test-support/scoll-into-view-mock.js

```diff
@@ -0,0 +1,11 @@
+import sinon from 'sinon';
+export default function setupScrollIntoView(hooks) {
+    hooks.beforeEach(function() {
+        this.scrollIntoViewMock = sinon.stub();
```

Comment:
I don't think this is the approach we want to take. If we mock the entire scrollIntoView modifier, then nothing is actually being tested. I think a better approach would be to stub the `scrollIntoView` method on the Element prototype: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

We can then record the value of `this` each time the method gets called to know which element it was called on, and return a method to the user that allows them to pass in a data-test selector to assert the correct element was called.

Reply:
wondering if running the test without browser might run into any issue if we are mocking Element.prototype.scrollIntoView.

Reply:
I think all our tests run with a headless browser anyway, so the APIs should still be available.